src/features/RendererRegistrar.cpp

```diff
@@ -19,17 +20,23 @@ namespace Features
 
         static SceneToRenderersMap renderers;
         INIT_RENDERER_MAP(renderers)
+			
+#ifdef _TEST
+			BEGIN_SCENE_TYPE(SCENE_TYPE_TEST0)
+				REGISTER_RENDERER(TestRenderer)
+			REGISTER_RENDERER(TestRenderer1)
+			// other renderer?
+			END_SCENE_TYPE()
 
-            BEGIN_SCENE_TYPE(SCENE_TYPE_TEST0)
-                REGISTER_RENDERER(TestRenderer)
-                REGISTER_RENDERER(TestRenderer1)
-                // other renderer?
-            END_SCENE_TYPE()
-
-            BEGIN_SCENE_TYPE(SCENE_TYPE_TEST1)
-                REGISTER_RENDERER(TestRenderer2)
-                // other renderer?
-            END_SCENE_TYPE()
+			BEGIN_SCENE_TYPE(SCENE_TYPE_TEST1)
+				REGISTER_RENDERER(TestRenderer2)
+			// other renderer?
+			END_SCENE_TYPE()
+			
```

Comment:
you may want to remove this trailing line.

Reply:
I did it.